lib/utils.pm

```diff
@@ -2044,4 +2045,25 @@ sub permit_root_ssh_in_sol {
     enter_cmd("[ `grep \"^PermitRootLogin *yes\" $sshd_config_file | wc -l` -gt 0 ] || (echo 'PermitRootLogin yes' >>$sshd_config_file; systemctl restart sshd)");
 }
 
+=head2 cleanup_disk_space
+    cleanup_disk_space();
+
+In fully_patch_system and minimal_patch_system, we'll create so many
+snapshots. which will cost large part of the disk space. We need to
+delete these snapshots before the migration.
+
+=cut
+
+sub cleanup_disk_space {
+    record_soft_failure "bsc#1192331", "Low diskspace on Filesystem root";
```

Comment:
This is now called unconditionally without checking the error condition first. You could call `script_run '[[ $(findmnt -n -D -r -o avail /) < 2G ]]'` with whatever limit you see feasible and only call `record_soft_failure` if that check returns true

Reply:
changed. thanks